src/masternodes/icxorder.h

```diff
@@ -28,8 +29,8 @@ class CICXOrder
     CScript ownerAddress; //address of token asset in case of DFC/BTC order
     CAmount amountFrom; // amount of asset that is selled
     CAmount amountToFill; // how much is left to fill the order
```

Comment:
`amountToFill` is `orderPrice - amountFrom` do we need separate field?

Reply:
amountToFill is a value how much is left to fill the order, and is updated as order is partially filled. If we don't want to keep what was the amount of the order when it was opened, then I can subtract from amountFrom directly!

Reply:
I don't get what you mean directly? You have price and the amount you have till now, amountToFill is always price - amount

Reply:
amountFrom is amount of asset selling - 100 DFI
orderPrice is price of 1 asset in opposite asset - 1 DFI = 0.00015 BTC
amountToFill is how much is left to fill - 100 DFI initially, when offer fills 35 DFI then amountToFill will be 65 DFI.

I can remove redundant data if we don't need to save initial amount of order (100 DFI) and then I can sub from amountFrom when successful trade is done. Also I could store orderPrice as price * amount so I have already calced amount how much it is in opposite asset and not calculating it every time.